util/util_test.go

```diff
@@ -6,6 +6,12 @@ import (
 	"testing"
 )
 
+func TestBinaryInstalled(t *testing.T) {
```

Comment:
Well if someone is keen to run `go test` they have to have `git` installed as well, so it should work on those systems. I think it'd be good to have an `osc` test as well, or maybe just get of both. Anyways, I made the test to diagnose why it didn't work in Windows - so in that sense the test already did its job :)